### DISCUSSION TITLE
Vibration Detection and Rejection from IMU Data (Project 231)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#65] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project #231](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Vibration%20Detection%20and%20Rejection%20from%20IMU%20Data)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by Odireleng-K** [comment]
Hi, I'm Odireleng, student at University of the Witwatersrand, Johannesburg. I'm interested in collaborating and working on this project

**Comment by swapnil259** [comment]
Hi, I am a PhD student at IIEST, India. I'm interested in working and collaborating on this project

**Comment by Ganishk** [comment]
Hi, I am an undergrad at the National Institute of Technology, Tiruchirappalli (NITT), India. and interested in working and collaborating on this project.

**Comment by musa170605** [comment]
I am a student of MS Control Automation and AI, interested in doing this project. from Coventry university.
